packages/web-react/src/components/UNSTABLE_Slider/UNSTABLE_Slider.tsx

```diff
@@ -0,0 +1,84 @@
+import classNames from 'classnames';
+import React, { ForwardedRef, forwardRef } from 'react';
+import { useStyleProps } from '../../hooks';
+import { SpiritSliderProps } from '../../types';
+import { HelperText, ValidationText, useAriaIds } from '../Field';
+import { SLIDER_DEFAULT_PROPS } from './constants';
+import { useSliderStyleProps } from './useSliderStyleProps';
+
+const defaultProps = {
+  ...SLIDER_DEFAULT_PROPS,
+};
+
+/* We need an exception for components exported with forwardRef */
+/* eslint no-underscore-dangle: ['error', { allow: ['_UnstableSlider'] }] */
+const _UnstableSlider = (props: SpiritSliderProps, ref: ForwardedRef<HTMLInputElement>) => {
+  const propsWithDefaults = { ...defaultProps, ...props };
+  const {
+    'aria-describedby': ariaDescribedBy,
+    helperText,
+    id,
+    label,
+    validationState,
+    validationText,
+    value,
+    min,
+    max,
+    step,
```

Comment:
I don't know if we sort these alphabetically.

Reply:
why not, sorted